_scripts/jaeger.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -ex
+
+
+if [ "$TRACING_JAEGER_EXPORTER_ENABLED" == "true" ]
```

Comment:
Oh sorry, I added some info below 'Other Information' in the PR. Unfortunately, it looks like the instructions were a bit incomplete. I just updated them to be specific about adding this environment variable to the .env file in fxa root. Please see these steps again and let me know if that works for you.

By the way, I like using .env for things like this because it's easy to gitingore, applies globally, and is easy apply when needed with `dotenv -- yarn start`. Tracing can collect quite a bit of data, so I am not sure we want to always be running it. This is the same approach I took for the sentry setup locally, and it has worked well minus a bit of fuss surrounding install dotenv. I'll just leave this up to devs though. If this does work out, I plan on adding the full documentation to our ecosystem docs.